### ISSUE TITLE
Trying again

### ISSUE THREAD

**Issue by AndyHoggANU** [aidanheerdegen/publish_cosima_data#14]
Hi @aidanheerdegen  - I am belatedly getting back to this data publishing pipeline.
First attempt revealed some errors like this:
```
Deleting following variables with a time dimension from ocean_grid.nc: ['time', 'drag_coeff']
Traceback (most recent call last):
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-19.07/bin/splitvar", line 10, in <module>
    sys.exit(main_argv())
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-19.07/lib/python3.6/site-packages/splitvar/cli.py", line 116, in main_argv
    main_parse_args(sys.argv[1:])
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-19.07/lib/python3.6/site-packages/splitvar/cli.py", line 110, in main_parse_args
    return main(parse_args(args))
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-19.07/lib/python3.6/site-packages/splitvar/cli.py", line 204, in main
    ds = xarray.decode_cf(ds)
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-19.07/lib/python3.6/site-packages/xarray/conventions.py", line 479, in decode_cf
    decode_coords, drop_variables=drop_variables, use_cftime=use_cftime)
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-19.07/lib/python3.6/site-packages/xarray/conventions.py", line 401, in decode_cf_variables
    stack_char_dim=stack_char_dim, use_cftime=use_cftime)
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-19.07/lib/python3.6/site-packages/xarray/conventions.py", line 306, in decode_cf_variable
    var = coder.decode(var, name=name)
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-19.07/lib/python3.6/site-packages/xarray/coding/times.py", line 419, in decode
    self.use_cftime)
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-19.07/lib/python3.6/site-packages/xarray/coding/times.py", line 90, in _decode_cf_datetime_dtype
    last_item(values) or [0]])
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-19.07/lib/python3.6/site-packages/xarray/core/formatting.py", line 99, in last_item
    return np.ravel(array[indexer]).tolist()
AttributeError: 'Array' object has no attribute 'tolist'
```
I am testing out whether this is due to the 19.07 update and will report back.

**Comment by AndyHoggANU** [comment]
Just to confirm - this is fixed by reverting back to 19.04, but it would be good if the unstable branch could be updated so that it works there too.

**Comment by aidanheerdegen** [comment]
This is a bug in xarray

https://github.com/pydata/xarray/issues/3215

Work-around is to set this environment variable:

```
NUMPY_EXPERIMENTAL_ARRAY_FUNCTION=0
```

I have submitted a PR with a code fix https://github.com/pydata/xarray/pull/3220
